src/include/common/types/value.h

```diff
@@ -97,25 +199,57 @@ class Value {
     std::unique_ptr<RelVal> relVal;
 };
 
+/**
+ * @brief NodeVal represents a node in the graph and stores the nodeID, name, properties of that
+ * node.
+ */
 class NodeVal {
 public:
+    /**
+     * @brief Constructs the NodeVal object with the given idVal and labelVal.
+     * @param idVal the nodeID value.
+     * @param labelVal the name of the node.
+     */
     KUZU_API NodeVal(std::unique_ptr<Value> idVal, std::unique_ptr<Value> labelVal);
-
+    /**
+     * @brief Constructs the NodeVal object from the other.
+     * @param other the NodeVal to copy from.
+     */
     KUZU_API NodeVal(const NodeVal& other);
-
+    /**
+     * @brief Adds a property with the given {key,value} pair to the NodeVal.
+     * @param key the name of the property.
+     * @param value the value of the property.
+     */
     KUZU_API void addProperty(const std::string& key, std::unique_ptr<Value> value);
-
+    /**
+     * @return returns all properties of the NodeVal.
```

Comment:
"returns all properties of the NodeVal." Should this be "all properties of the NodeVal." ?